### PR TITLE
- add warnings if file name is too long

### PR DESCRIPTION
**Pull request by vwaurich**

### CODE REVIEW

File: Compiler/runtime/systemimpl.c [hunk 1]
```diff
@@ -2892,15 +2892,18 @@ int SystemImpl__covertTextFileToCLiteral(const char *textFile, const char *outFi
...
   char buffer[512];
   char obuffer[1024];
   fin = fopen(textFile, "r");
   if (!fin) {
     goto done;
   }
+  errno = 0;
   fout = fopen(outFile, "w");
   if (!fout) {
+    printf("SystemImpl__covertTextFileToCLiteral could not create file. The total file name might be too long. Error %d \n", errno);
```
Comment (sjoelund) [inline Compiler/runtime/systemimpl.c]: Why not use `getError()` to return error messages? (And `strerror(errno)` to return the error-message set by fopen?)

**Comment by OpenModelica-Hudson** [comment]
The job failed to compile; for details, see [OpenModelica_TEST_PULL_REQUEST 2016-09-29_13-37-11](https://test.openmodelica.org/hudson//job/OpenModelica_TEST_PULL_REQUEST/3164/console).

File: Compiler/runtime/systemimpl.c (hunk 1 above)
Comment (vwaurich) [inline Compiler/runtime/systemimpl.c]: ok?

File: Compiler/runtime/systemimpl.c [hunk 2]
```diff
@@ -2903,7 +2903,7 @@ int SystemImpl__covertTextFileToCLiteral(const char *textFile, const char *outFi
   errno = 0;
   fout = fopen(outFile, "w");
   if (!fout) {
-    printf("SystemImpl__covertTextFileToCLiteral could not create file. The total file name might be too long. Error %d \n", errno);
+    fprintf(stderr , "SystemImpl__covertTextFileToCLiteral failed: %s Maybe the total file name is too long.", strerror(errno));
```
Comment (sjoelund) [inline Compiler/runtime/systemimpl.c]: This could still use the error module. If you print to stdout or stderr, the user cannot see it when scripting OMC

**Comment by vwaurich** [comment]
Another try. Is this the correct error module?

**Comment by sjoelund** [comment]
Yes, but the commits should be squashed into a single commit (and no commit message title should ever start with a -)

**Comment by vwaurich** [comment]
Alright

**Pull request closed without merging**